src/quo2/components/messages/gap.cljs

```diff
@@ -20,12 +21,18 @@
   [key]
   (get-in themes [(theme/get-theme) key]))
 
+(defn js-require
```

Comment:
Why do we need to use a different resource if the component is running under the Jest umbrella? Have you tried using `with-redefs` in tests that need different resources, instead of changing the production code?

Reply:
hmm, it's not the test directly that needs the resources but some file that\s a dependency of the test file. I think we just ultimately need to configure something and it should resolve this.

Reply:
> it's not the test directly that needs the resources but some file that\s a dependency of the test file.

I still don't understand :man_shrugging: Which file are you referring to that requires this path change when running via Jest? I'm just trying to understand the root cause, because it's not apparent from the diff alone.